### PR TITLE
Update to support Flounder/6.0.0

### PR DESCRIPTION
**Pull request by sebd71**
- remove landscape mode, back to portrait mode
- update to support new WindowManager
- support qemu

Change-Id: I30fddfb2ec0ddbe96a8f30f75d800fff62aa08bc
Signed-off-by: Sebastien Douheret <sebastien.douheret@iot.bzh>

### CODE REVIEW

**Comment by sebd71** [comment]
I'm not sure we should drop eel support as it's working (maybe with some limitations).
I would be more in flavour to move changes done for eel in a branch name eel, keep changes for FF in flounder branch and use master branch to handle WAM on current/master AGL version.

File: recipes-wam/wam/wam.bb [hunk 1]
```diff
@@ -5,14 +5,17 @@ LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/Apache-2.0;md5=89aea4e17d99a7ca
 
 inherit qmake5
 
-DEPENDS = "qtbase glib-2.0 chromium53 wayland-ivi-extension virtual/libhomescreen libwindowmanager"
+DEPENDS = "qtbase glib-2.0 chromium53 wayland-ivi-extension libhomescreen libwindowmanager"
```
Comment (jkim-julie) [inline recipes-wam/wam/wam.bb]: Hi Sebastien,
When I tried with this patch, I needed to keep 'virtual/' for libhomescreen. Not sure it only happened on my environment. Maybe is there something I missed?

Comment (sebd71) [reply recipes-wam/wam/wam.bb]: Hello,
sorry for the late response.
If I'm correct, following this change (https://gerrit.automotivelinux.org/gerrit/#/c/16489/) we no longer use virtual/libhomscreen.

Which method and command line did you use to build AGL : based on repo or based on prepare_meta ?
You must use either flounder branch or latest master (knowing that master is currently very similar to flounder, just branched few days ago).

Anyway, here are commands line to setup your build :

```
# using repo method
repo init -b flounder -m flounder_6.0.0.xml -u https://gerrit.automotivelinux.org/gerrit/AGL/AGL-repo
git clone https://github.com/iotbzh/meta-agl-lge.git && (cd meta-agl-lge && git checkout flounder)
source meta-agl/scripts/aglsetup.sh -m m3ulcb agl-devel agl-netboot agl-appfw-smack agl-demo agl-localdev agl-audio-4a-framework agl-hmi-framework agl-demo-wam

# using prepare_meta
prepare_meta -f flounder-wam -o /xdt -l $MIRROR -e wipeconfig -e cleartemp -e rm_work -t qemux86-64 agl-devel agl-netboot agl-appfw-smack agl-demo agl-localdev agl-audio-4a-framework agl-hmi-framework agl-demo-wam
bitbake agl-demo-platform-wam

```

Comment (jkim-julie) [reply recipes-wam/wam/wam.bb]: Hi, thanks. It works.

**Pull request merged**